.github/workflows/website.yml

```diff
@@ -4,6 +4,10 @@ on:
   push:
     branches:
       - master
+    paths:
+      - 'docs/**'
+      - 'website/**'
+      - 'couscous.yml'
```

Comment:
this should also include `.github/workflows/website.yml` so that changing the workflow file also triggers a run on next push to master.

Reply:
added